### DISCUSSION TITLE
Discharging and discharging reversed representation

### DISCUSSION THREAD

**Discussion by JeNdRiKHemmEn** [Louisvdw/dbus-serialbattery#523] [Q&A] [answered]
Hello dear Louis,
I hooked up my Daly BMS to my Victron Multiplus GX just as Andy from the off-grid garage showed. Has worked well so far. Unfortunately, the display of loading and unloading is reversed on my dashboard. So it shows unloading when it is loading and vice versa. In the cumulative display of how many kWh was charged into the battery or removed from the battery, the display is correct again. So it's just a small display error.

https://www.youtube.com/watch?v=sHIlP2oJ3DU

**Comment by mr-manuel** [comment, marked as answer]
Have you checked this setting?

https://github.com/Louisvdw/dbus-serialbattery/blob/3685d15a5d1ba801867bbdb98f57fd805ea2c24a/etc/dbus-serialbattery/utils.py#L136-L137

**Comment by JeNdRiKHemmEn** [comment]
Thank you that helped. 
Thank you for the really quick help


Mit freundlichen Grüßen 
M.Sc. Ing. Jendrik Hemmen Elektrofachkraft 
Wecholder Straße 37
28277 Bremen
0163-9886868 

    Am Dienstag, 11. April 2023 um 13:32:33 MESZ hat Manuel ***@***.***> Folgendes geschrieben:  
 
 


Have you checked this setting?

https://github.com/Louisvdw/dbus-serialbattery/blob/3685d15a5d1ba801867bbdb98f57fd805ea2c24a/etc/dbus-serialbattery/utils.py#L136-L137
